src/lib/api/conversions.ts

```diff
@@ -0,0 +1,54 @@
+import { RemoteItem } from '@dcl/builder-client'
```

Comment:
What do you think of renaming this file to `transformations` or something of the sort? conversions is usually used for units of measurements or monetary values